### DISCUSSION TITLE
Github not accepting my school ID

### DISCUSSION THREAD

**Discussion by c9long** [community/community#21132] [GitHub Education]
Hi all,

Like many others posting on the Education board, I can't seem to get past Github's (seemingly too stringent) automatic verification system when applying for the Student Developer Pack. I attend the University of Waterloo in Canada and our student ID cards aren't dated with a "valid until" section. Nevertheless, I was able to move furthest when submitting a photo of it, back and front, receiving the following error after choosing to ignore a warning:
![image](https://user-images.githubusercontent.com/109810526/180609508-18756e08-6c04-4715-84d6-508cf4c5b363.png)

It felt quite frustrating that despite giving me the option to skip the warning, Github still rejected my application on an automated basis. I have also tried using my school's letter of enrolment, unofficial transcript, and a screenshot of the school portal, all of which are digital. I am met with the following error message:
![image](https://user-images.githubusercontent.com/109810526/180609590-52f37a08-5383-490d-b4be-cf3bb4414b2f.png)

Furthermore, when I select the specific type of document I am submitting, I am met with one more error message telling me I MUST submit a school ID (aka a student card) **despite giving me the option to choose other methods of proof**:
![image](https://user-images.githubusercontent.com/109810526/180609727-adb32763-5f50-4c76-a93b-9c2548403eb6.png)

You might suggest that I try printing my documents out and taking a picture of them, or taking a picture of my screen. I have already attempted to do so, to the same error message. Furthermore, my teammates working on the same project from the same university were able to have their screenshots approved without any issue.

Github also does not offer any direct support chat, hotline, or email. I had submitted a ticket about 24 hours ago to their old community forum, and have now found this discussion board. The lack of human support and heavy reliance on a ML algorithm has made the whole process and my experience with Github extremely disappointing.

I would greatly appreciate any help from Github staff with this issue.

tl;dr Student Dev Pack auto-blocking all of my legitimate, up-to-date, concurrent proofs of school ID.

-c9long

**Comment by eddiebauer7** [comment]
It's the worst verification system ever, honestly.

**Reply by c9long** [reply]
I wouldn't go so far as to say that, being completely objective, I've seen far worse lol. But I share your frustration with it, it could be better, though I also understand Github is severely understaffed to handle requests manually.

**Comment by pidoshva** [comment]
Just request a Proof of Enrollment through your school's website. That's what I did and it worked for me.
Unofficial transcript should work too.

**Reply by c9long** [reply]
![image](https://user-images.githubusercontent.com/109810526/180896657-ba3b4005-9e2d-43ec-a5c3-e1fe8143c29a.png)

**Reply by pidoshva** [reply]
Did you try to include a screenshot of an actual section with date on your transcript or enrollment? Because pdfs did not work for me as well...

**Reply by c9long** [reply]
Yes, they don't actually accept PDFs, only JPEG's and PNG's. I've tried both file formats, a variety of different screenshot sizes, emphasis on the date, etc. I also don't understand why this: 
![image](https://user-images.githubusercontent.com/109810526/180897217-ce41b49d-02bc-42a3-b1a2-f995c9112bc9.png)
is even a reason for rejection when they literally ask for documents that most schools only offer electronically now.

**Reply by pidoshva** [reply]
Weird thing is that I was getting the same error messages when was trying to verify my enrollment a couple months ago. I tried a couple days ago with just a screenshot of my enrollment documents that shows the date and got accepted automatically in seconds. 
GitHub automation system definitely needs some changes.

**Comment by joshuawalker** [comment]
Hey @c9long—the Education team has your support ticket and they'll get back to you as soon they can.

**Reply by c9long** [reply]
Thanks for the update :)

**Reply by c9long** [reply]
Hi @joshuawalker, any chance the support team can take a look at my ticket again? The help I received was lacklustre and merely suggested what the automated replies have been suggesting (to take photos with my device camera instead). Here is a screenshot of me having done exactly that to the same annoying error message:
![image](https://user-images.githubusercontent.com/109810526/180907290-7b7d87a1-4061-4ed3-b14c-b5f6d02774ca.png)

**Comment by SohailSayed** [comment]
Hey were you able to get approved finally? I'm from Waterloo too and my application is currently pending

**Comment by shivangshandilya** [comment]
So I was reverifying myself on GitHub portal and when asked for the proof I give my college ID but it says its already been used before. I dont have any other means to verify myself, idk what to do. Everything is clear on the ID then why does it ask this?
![image](https://github.com/community/community/assets/101946115/afa0b58a-528e-41ef-a528-156ee2e43f8d)
